### PR TITLE
Release 2.0.2+rev1

### PR DESCRIPTION
**Pull request by agherzan**

### CODE REVIEW

**Comment by lifeeth** [comment]
The image does not show up online on the dashboard

**Comment by agherzan** [comment]
@telphan will take a look on this.

**Comment by floion** [comment]
Any update on this @telphan ?

**Comment by telphan** [comment]
@agherzan can we have this rebased and then I can test?

**Comment by floion** [comment]
@telphan you can rebase this yourself.

**Comment by telphan** [comment]
Hence me deleting my comment, as I realized that.

**Comment by willnewton** [comment]
We may as well bump to 2.0.2 while we are doing the update.

**Comment by telphan** [comment]
@willnewton Ack

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +20,7 @@ Change log
 
 * Bump tests/autohat to current HEAD [Andrei]
 * Bump resin-yocto-scripts to current HEAD [Andrei]
-* meta-resin: Bump to 2.0.0-beta12 [Andrei]
+
```
Comment (michal-mazurek) [inline CHANGELOG.md]: Is this an empty newline? Doesn't look OK.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Change log
 -----------
 
+# v2.0.2+rev1 - 2017-04-13
```
Comment (michal-mazurek) [inline CHANGELOG.md]: This was nearly a month ago, shan't we update the date?

File: CHANGELOG.md (hunk 1 above)
Comment (michal-mazurek) [inline CHANGELOG.md]: Also I don't get why we're removing this old entry.

**Pull request merged**